header-validator/src/source.test.ts

```diff
@@ -610,6 +610,84 @@ runAll(validateSource, [
       },
     ],
   },
+  {
```

Comment:
Can we add a test for expiry < 86400 ? Unless we have another condition that makes this impossible ?

IIUC this would round down to zero which based on the function name `roundAwayFromZero` might not be desirable.

Reply:
We already have a test for this in the clamping section.